src/components/atoms/UserAvatar/UserAvatar.scss

```diff
@@ -49,19 +65,11 @@
     }
   }
 
-  &--small {
-    width: 25px;
-    height: 25px;
-  }
-
-  &--medium {
-    height: 40px;
-    width: 40px;
-  }
-
-  &--large {
-    height: 54px;
-    width: 54px;
+  @each $modifier, $size in $avatar-sizes-map {
+    &--#{$modifier},
+    &--#{$modifier} img {
+      @include square-size($size);
+    }
```

Comment:
The syntax is shorter but the readability is not better. We should choose one and keep it consistent, I haven't seen this syntax in the project so far. And we need to communicate these things before applying them. If I start adding my own standards and styles with every PR you guys will hate me by the end of the week 😆

Reply:
>  I haven't seen this syntax in the project so far. 

https://github.com/sparkletown/sparkle/blob/staging/src/scss/constants.scss contains mixins, functions, interpollation...

Standard SCSS syntax really https://sass-lang.com/documentation 
and my view is if we don't use it as intended to make few things DRY and robust, might as well move to back to CSS.


> And we need to communicate these things before applying them.

PR review process is communication, PR merge is application, so we're good I think.

> you guys will hate me by the end of the week

Not really. Not a fan of setting things in stone.